CONTRIBUTING.rst

```diff
@@ -37,6 +37,18 @@ from any directory in your system with::
 
     jupyter server
 
+For connecting the frontend app with the server, you need to run the following command::
+
+    pip install nbclassic
+
+Once it is completed, you can connect the server with the frontend app with the following command::
+
+    jupyter nbclassic --ServerApp.log_level="CRITICAL"
+
+The following log levels are supported:
```

Comment:
```suggestion
The following log levels are supported:

```

Reply:
I am using a vs code extension  viewing .rst files. Is there any specific extension, I could use or something else that lets me preview the changes. I didn't notice any formatting errors with the extension I am using. Thank you!